packages/app-builder/src/repositories/ScenarioRepository.ts

```diff
@@ -45,10 +46,15 @@ export interface ScenarioRepository {
     scenarioId: string;
   }): Promise<ScenarioIterationSummary[]>;
   validate(args: { iterationId: string }): Promise<ScenarioValidation>;
-  validateScenarioIterationTrigger(args: {
+  validateTrigger(args: {
```

Comment:
pretty nice, that's a good usecase of having an abtraction over the REST backend API in the front repo 👍